chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/BrokenSurveillanceRulesCreatorJob.java

```diff
@@ -50,47 +49,46 @@
  */
 @DisallowConcurrentExecution
 public class BrokenSurveillanceRulesCreatorJob extends QuartzJob {
-    private static final Logger LOGGER = LogManager.getLogger(BrokenSurveillanceRulesCreatorJob.class);
+    private static final Logger LOGGER = LogManager.getLogger("brokenSurveillanceRulesCreatorJobLogger");
```

Comment:
This statically declared logger didn't have to get moved into getLogger... why?

Reply:
Other code called by the job, should be logged to the job's log file.  That is the need for the getter/setter for the logger in the shared files.  I did not go down to the Manager and DAO level, but that can be looked into.  With the changes I made, I captured ~95% of all logger statements into the specific job's logger file.

Reply:
Ok - I guess just keep it in mind if we see logging in the main Tomcat log that isn't the result of user activity on the CHPL. I feel like it could get all mixed up in there and be confusing when trying to debug something but seems like it would be a lot of work to change. It is probably not even an issue until we start logging more stuff. :)